assets/src/web-stories-block/block/api/wpAdapter.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+const get = (path, options = {}) => apiFetch({ path, ...options });
+
+export default { get };
```

Comment:
What is the point of this wrapper?

Reply:
I've kept the file or wrapper function in case we need to understand where all the dependencies arose from while deduping the changes, later on, also this will help to modify it later in case we need to incorporate any changes.

Reply:
> I am guessing that DEPRECATED_THEME is used every becuase of the new design system is that correct?

Correct, we are using the web story dashboard design system.

Reply:
I would say remove the wrapper, seems pointless.

Reply:
removed.